SeaPublicWebsite/Services/Cookies/CookieService.cs

```diff
@@ -90,6 +90,6 @@ public void SetCookie<T>(HttpResponse response, string cookieName, T cookie)
         response.Cookies.Append(
             cookieName,
             cookieString,
-            new CookieOptions {Secure = true, SameSite = SameSiteMode.Lax, MaxAge = TimeSpan.FromDays(Configuration.DefaultDaysUntilExpiry)});
+            new CookieOptions {Secure = true, SameSite = SameSiteMode.Lax, MaxAge = TimeSpan.FromDays(Configuration.DefaultDaysUntilExpiry), HttpOnly = true});
```

Comment:
The report makes it look like all the cookies had `httponly` already, and it's just that the built in anti forgery token didn't have `secure`. Have you checked for that?